core/src/main/resources/hudson/cli/Messages.properties

```diff
@@ -113,3 +113,12 @@ WaitNodeOfflineCommand.ShortDescription=Wait for a node to become offline.
 
 CliProtocol.displayName=Jenkins CLI Protocol/1 (unencrypted)
 CliProtocol2.displayName=Jenkins CLI Protocol/2 (transport encryption)
+
+DisablePluginCommand.ShortDescription=\
```

Comment:
nit: why the `\`?

Reply:
To introduce a new line, used in many other `ShortDescription` messages.

Reply:
Not nit for me, just a typo that should be correct 👍

Reply:
Initially I also put a negative comment on that but if you look at the file, it's full of this kind of stuff. I imagine that compared to HTML our CLI is not eating the initial blank lines ;)

Reply:
Indeed. It's just a matter of taste